lib/full-site-editing/templates-utils.php

```diff
@@ -41,81 +75,27 @@ function gutenberg_render_templates_lists_custom_column( $column_name, $post_id
 	}
 
 	if ( 'status' === $column_name ) {
-		$post_status = get_post_status( $post_id );
-		// The auto-draft status doesn't have localized labels.
-		if ( 'auto-draft' === $post_status ) {
-			echo esc_html_x( 'Auto-Draft', 'Post status', 'gutenberg' );
-			return;
-		}
+		$post_status        = get_post_status( $post_id );
 		$post_status_object = get_post_status_object( $post_status );
 		echo esc_html( $post_status_object->label );
 		return;
 	}
 
 	if ( 'theme' === $column_name ) {
-		$terms = get_the_terms( $post_id, 'wp_theme' );
-		if ( empty( $terms ) || is_wp_error( $terms ) ) {
-			return;
-		}
-		$themes        = array();
-		$is_file_based = false;
-		foreach ( $terms as $term ) {
-			if ( '_wp_file_based' === $term->slug ) {
-				$is_file_based = true;
-			} else {
-				$themes[] = esc_html( wp_get_theme( $term->slug ) );
+		$terms  = gutenberg_parse_wp_theme_terms( $post_id );
+		$themes = array();
+		foreach( $terms['themes'] as $term_theme ) {
+			$theme = wp_get_theme( $term_theme );
+			if ( $theme->exists() ) {
+				$themes[] = esc_html( $theme );
 			}
 		}
 		echo implode( '<br />', $themes );
-		if ( $is_file_based ) {
-			echo '<br />' . __( '(Created from a template file)', 'gutenberg' );
+		if ( $terms['is_original'] ) {
+			echo '<br />' . __( '(Template file — not customized)', 'gutenberg' );
+		} else if ( $terms['is_file_based'] ) {
+			echo '<br />' . __( '(Originated from a template file)', 'gutenberg' );
```

Comment:
Maybe this could change to 'Template file - customized'? Might be nice to have it consistent with the inverse option